W-16993325-pce4-documentation-kt (#344)
mulesoft/docs-private-cloud: modules/ROOT/pages/install-workflow.adoc

= Installing Anypoint Platform Private Cloud Edition (Anypoint Platform PCE)
:page-aliases: prereq-gravity-check.adoc

The process to install Anypoint Platform Private Cloud Edition (Anypoint Platform PCE), consist of the following steps:

. Setting up and verifying your infrastructure
. Performing a pre-installation verification
. Installing Anypoint Platform PCE

[NOTE]

> You must contact MuleSoft Professional Services prior performing the pre-installation verification and installation of Anypoint Platform PCE, as they are involved in these processes. 

Reviewer: ```suggestion
Contact MuleSoft Professional Services before performing the pre-installation verification and installation of Anypoint Platform PCE because they are involved in these processes. 
```

== Set Up and Verify your Infrastructure

> Before you install Anypoint Platform PCE, your infrastructure team must confirm that they installed and configured correctly the infrastructure for your organization following the requirements as described in xref:install-checklist.adoc[Anypoint Platform PCE Prerequisites].

Reviewer: ```suggestion
Before you install Anypoint Platform PCE, ensure your infrastructure team installed and configured the infrastructure for your organization following the requirements described in xref:install-checklist.adoc[Anypoint Platform PCE Prerequisites].
```

      fileSystemDNS: <PLACEHOLDER_URL>
      firstUserAccount:
        organizationName: Test Org
        username: <PLACEHOLDER_USERNAME>
        email: <PLACEHOLDER_EMAIL>
        password: <PLACEHOLDER_PASSWORD>
      platformCertificate: /usr/src/app/workdir/cert.crt
      platformCertificateKey: /usr/src/app/workdir/cert.key
    amvInstallConfiguration:
      enabled: false
      influxdbLicensePath: ''
    storageConfiguration:
      pceobjectstore:
        type: persistentVolumeClaim
        storageClass: local-path
      stolon:
        type: persistentVolumeClaim
        storageClass: local-path
      stolonAmv:
        type: persistentVolumeClaim
        storageClass: local-path
      influxdb:
        type: persistentVolumeClaim
        storageClass: local-path
      logstash:
        type: persistentVolumeClaim
        storageClass: local-path
      cassandra:
        type: persistentVolumeClaim
        storageClass: local-path
----
[start=3]
. Obtain the `license.pem` file from the MuleSoft Professional Services team.
. Verify that the cluster satisfies the prerequisites by running the following command against your `input.yaml` file and the `license.pem` file:

> `./pcectl pre-install-check --input-file/path/to/input.yaml --license-file /path/to/license.pem`

Reviewer: ```suggestion
. Verify that the cluster satisfies the prerequisites by running the following command against your `input.yaml` file and the `license.pem` file:
+
`./pcectl pre-install-check --input-file/path/to/input.yaml --license-file /path/to/license.pem`
```
I think you need this to ensure the command shows indented properly. Please confirm in a local build.

== Install Anypoint Platform PCE 

> After your infrastructure is correctly set up and you performed a pre-installation verification, the MuleSoft Professional Services will assist you with the installation of Anypoint Platform PCE.

Reviewer: ```suggestion
After your infrastructure is correctly set up and you performed a pre-installation verification, MuleSoft Professional Services will assist you with the installation of Anypoint Platform PCE.
```
